app.py

```diff
@@ -28,5 +28,6 @@ def hello_world():
         apobj.notify(
             title=data['title'],
             body=data['body'],
+            notify_type=data['notify_type'],
```

Comment:
I guess this fails if the parameter is set, so this change would not be backward compatible. Using `data.get('notify_type', None)` instead probably fixes this?

Reply:
Good catch! Fixed it